### DISCUSSION TITLE
Number of significant components in decomposition

### DISCUSSION THREAD

**Discussion by lscaldas** [hyperspy/hyperspy#3442] [Q&A]
The elbow analysis of the scree plot of the core_shell.hdf5 dataset gives a result of 4 PCA components (SVD), but in David Rossouw et al. paper, it is said there are 3 components. Why?

Is there any data pre-treatment or a different algorithm, perhaps centering the data, that reduced the number of components estimated by the elbow algorithm? I am trying to reproduce the same scree plot as in the paper, but so far, I haven't been able to. Could someone help me with this? The reason I am trying to understand this dataset is that in my own data, the learning_results.number_significant_components consistently gives one more PCA component than my intuition would suggest. Therefore, I am trying to apply this to a well-known dataset that has already been analyzed and published.

**Comment by CSSFrancis** [comment]
Hi @lscaldas sorry this question fell by the wayside 😅  Can you maybe plot the scree plot?  Although there are a couple of calculations for the knee of a scree plot, many times people tend to play with their data and see what number of components give the best result.  

Sometimes people will have some previous knowledge on the number of components and explicitly force a number of components. I realize that might not be the most satisfying answer.
